src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -127,17 +114,15 @@ export class AddBulkConcepts extends Component {
 
   render() {
     const dictionaryName = localStorage.getItem('dictionaryName');
-    const { conceptIds, openResultModal, otherSelected } = this.state;
+    const { conceptIds, openResultModal } = this.state;
     const {
       match: {
         params: {
           type, typeName, collectionName, language,
         },
       },
       sourceConcepts,
-      conceptSources,
```

Comment:
How is this related to removing other?

Reply:
This was used by the others search field to filter the source names by the input data provided. This is shown on line 228 that I had to clear.